python/openassetio/hostAPI/transactions.py

```diff
@@ -173,16 +174,15 @@ def freezeManagerState(self, context):
         return "%i_%s" % (context.actionGroupDepth, token)
 
     @auditApiCall("Transactions")
-    def thawManagerState(self, token, context):
+    def thawManagerState(self, token):
         """
-        Restores the @ref manager_state in the supplied Context so that
-        it represents the context as previously frozen.
+        Restores the @ref manager_state returning a Context that is
+        associated with the same state as the context that was
+        previously frozen using freezeManagerState.
```

Comment:
[consider][minor]
```suggestion
        previously frozen using `freezeManagerState`.
```
or
```suggestion
        previously frozen using @ref freezeManagerState.
```